src/imitation/util/networks.py

```diff
@@ -259,3 +278,99 @@ def build_mlp(
     model = nn.Sequential(layers)
 
     return model
+
+
+def build_cnn(
+    in_channels: int,
+    hid_channels: Iterable[int],
+    out_size: int = 1,
+    name: Optional[str] = None,
+    activation: Type[nn.Module] = nn.ReLU,
+    kernel_size: int = 3,
+    stride: int = 1,
+    padding: Union[int, str] = "same",
+    dropout_prob: float = 0.0,
+    squeeze_output: bool = False,
+    normalize_input_layer: Optional[Type[nn.Module]] = None,
+) -> nn.Module:
+    """Constructs a Torch CNN.
+
+    Args:
+        in_channels: number of channels of individual inputs; input to the CNN will have
+            shape (batch_size, in_size, in_height, in_width).
+        hid_channels: number of channels of hidden layers. If this is an empty iterable,
+            then we build a linear function approximator.
+        out_size: required size of output vector.
```

Comment:
Required is a bit confusing given it has a default value (so is not required for the user to specify).

Reply:
Will fix that here and in `build_mlp`.